js/cronsync.js

```diff
@@ -250,11 +252,11 @@ CronSync.prototype = {
     // sync if it is sufficiently recent.
 
     // - Initiate a sync of the folder covering the desired time range.
-    this._LOG.syncAccount_begin(account.id);
-    this._LOG.syncAccountHeaders_begin(account.id, null);
+    logic(scope, 'syncAccount_begin');
```

Comment:
Yeah, async/await do seem a lot better than trying to do regexp pairing.  So colons just as a an idiomatic naming convention then.